syntax = "proto3";

message ConnectionMessage {
  int32 id = 1;
  int32 person_id = 2;
  string coordinate = 3;
  string creation_time = 4;
}

service ConnectionService {
    rpc Create(ConnectionMessage) returns (ConnectionMessage);
}